### ISSUE TITLE
issues with undo shortcut ctrl+z on windows10

### ISSUE THREAD

**Issue by neo-backwards-one** [cwensley/pablodraw#35]
# PabloDraw3.3.0-alpha1 on windows 10 (1909)

sometimes when trying to undo via shortcut ctrl+z instead of undoing it adds new characters to the drawing as shown in the picture provided.

![03](https://user-images.githubusercontent.com/78374580/106755363-570e0f00-662e-11eb-8b2d-080966f3dace.png)

one way of fixing this is selecting edit from the menu and closing it again right after. ctrl+z then works again, but i havent measured how long that persists.

**Comment by cwensley** [comment]
Hm, yeah there's something wrong going on with shortcuts on windows.  Thanks for reporting the issue!

**Comment by ericolito** [comment]
On Linux as well!

**Comment by neo-backwards-one** [comment]
> Hm, yeah there's something wrong going on with shortcuts on windows. Thanks for reporting the issue!

thats true this applies for other shortcuts as well. also on pablo2 one would hit backspace while shift key is still pressed. ever since pablo3 this acts like a shortcut to add the character "" which i'm probably not copying correctly here. why this is annoying for me is because i use a lot of "/"'s in my pictures and my pinky usually just sits on the shift key.

**Comment by cwensley** [comment]
Thanks for the additional information, @neo-backwards-one.  I'll try to resolve that one as well.

**Comment by cwensley** [comment]
@neo-backwards-one The original issue with ctrl+z should now be fixed with [3.3.0-beta1](https://github.com/cwensley/pablodraw/releases/tag/3.3.0-beta1), I still have to work on the shift+backspace issue.
